lib/WorkflowTaxprofiler.groovy

```diff
@@ -93,7 +93,7 @@ class WorkflowTaxprofiler {
                 params.run_bracken ? "Bracken (Lu et al. 2017)," : "",
                 params.run_kraken2 ? "Kraken2 (Wood et al. 2019)," : "",
                 params.run_krakenuniq ? "KrakenUniq (Breitwieser et al. 2018)," : "",
-                params.run_metaphlan3 ? "MetaPhlAn3 (Beghini et al. 2021)," : "",
+                params.run_metaphlan ? "MetaPhlAn3 (Beghini et al. 2021)," : "",
```

Comment:
Thanks for this correction.
`params.run_metaphlan ? "MetaPhlAn (Blanco-Miguez et al. 2023)," : "",`

Reply:
Oops, how did I miss that :face_exhaling: